scvi_in_R.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>add counts to getassaydata</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>